test/plugins/validation/oas3/security-definitions-ibm.js

```diff
@@ -0,0 +1,235 @@
+const expect = require('expect');
+const {
+  validate
+} = require('../../../../src/plugins/validation/oas3/semantic-validators/security-definitions-ibm');
+
+describe('it should have a type of `apiKey`,`http`,`oauth2`, `openIdConnect`', function() {
+  it('should have a type', function() {
+    const spec = {
+      components: {
+        securitySchemes: {
+          SecuritySchemeModel: {
+            scheme: 'basic'
+          }
+        }
+      }
+    };
+
+    const res = validate({ resolvedSpec: spec });
+    expect(res.errors.length).toEqual(1);
+    expect(res.warnings.length).toEqual(0);
+    expect(res.errors[0].message).toEqual(
+      "'type' must be defined as a string and is required for path: securitySchemes.SecuritySchemeModel"
+    );
+  });
+  it('type can only be `apiKey`, `http`, `oauth2`, `openIdConnect`', function() {
+    const spec = {
+      components: {
+        securitySchemes: {
+          SecuritySchemeModel: {
+            type: 'wrong type',
+            scheme: 'basic',
+            descriptions: 'example text'
```

Comment:
Just for clarity, this should be `description`, as `descriptions` is not an actual field in the spec